wiki/Guides/Compressing_files/en.md

```diff
@@ -43,8 +47,14 @@ This section will show you how to remove audio from videos using [Handbrake](htt
 
 ![Removing audio tracks from the audio tab in Handbrake](img/removeaudio-handbrake.png "Removing the audio tracks")
 
-4. Go into the `Video` tab and make sure the video codec is set as `H.264 (X264)`. Change the `Constant Quality` to between 20–25. Smaller values will produce larger file sizes but with a higher video quality.
-5. If you are willing to spend more time encoding, change the `Encoder Preset` under `Encoder Options`. Slower presets deliver better video quality and may also reduce video file size, but do not go down to placebo as it takes much longer than `VerySlow` for very little improvement in quality. Set the framerate to be the same as the source and set it to constant framerate.
+4. Go into the `Video` tab and make sure to set the following settings:
+   - `Video Encoder` set to `H.264 (x264)` to encode in the H.264 format using the x264 encoder.
+   - `Framerate (FPS)` set to `Same as source` with `Constant Framerate` selected.
+   - `Constant Quality` set to a value of between 20 to 25, with smaller values resulting in larger, higher quality files.
+   - `Encoder Profile` set to `High` to use the H.264 High profile.
```

Comment:
Actually, after doing further tests myself, the difference between High and Main profile is minimal especially at the 720p maximum resolution osu! supports. So probably it can be removed entirely.